samples/net/https_client/boards/nrf54l15dk_nrf54l15_cpuapp.conf

```diff
@@ -0,0 +1,87 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Wi-Fi
+CONFIG_WIFI=y
+CONFIG_WIFI_NRF70=y
+
+# WPA supplicant
+CONFIG_WIFI_NM_WPA_SUPPLICANT=y
+CONFIG_WIFI_NM_WPA_SUPPLICANT_LOG_LEVEL_INF=y
+
+# Networking
+CONFIG_NET_L2_ETHERNET=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_IPV6=y
+CONFIG_NET_UDP=y
+CONFIG_NET_TCP=y
+CONFIG_NET_DHCPV4=y
+CONFIG_DNS_RESOLVER=y
+CONFIG_DNS_RESOLVER_AI_MAX_ENTRIES=4
+
+# NET sockets
+CONFIG_NET_UDP=y
+CONFIG_NET_TCP=y
+CONFIG_NET_MAX_CONN=6
+CONFIG_NET_MAX_CONTEXTS=5
+CONFIG_NET_SOCKETS_POLL_MAX=8
+CONFIG_NET_SOCKETS_OFFLOAD=n
+CONFIG_NET_DHCPV4=y
+CONFIG_NET_CONTEXT_SNDTIMEO=y
+CONFIG_NET_CONTEXT_RCVTIMEO=y
+CONFIG_NET_MAX_CONTEXTS=3
+
+# NET buffers
+CONFIG_NET_PKT_TX_COUNT=6
+CONFIG_NET_PKT_RX_COUNT=6
+CONFIG_NET_BUF_TX_COUNT=12
+CONFIG_NET_BUF_RX_COUNT=6
+CONFIG_NRF70_RX_NUM_BUFS=6
+CONFIG_NET_PKT_BUF_RX_DATA_POOL_SIZE=4096
+CONFIG_NET_PKT_BUF_TX_DATA_POOL_SIZE=4096
+CONFIG_NRF70_MAX_TX_AGGREGATION=1
+
+# Kernel options
+CONFIG_POSIX_NETWORKING=y
+CONFIG_POSIX_MAX_FDS=21
+
+# Shell
+CONFIG_SHELL=y
+CONFIG_SHELL_CMDS_RESIZE=n
+CONFIG_NET_L2_WIFI_SHELL=y
+
+# TLS
+CONFIG_NET_SOCKETS_SOCKOPT_TLS=y
+CONFIG_NET_SOCKETS_TLS_MAX_CONTEXTS=1
+CONFIG_NRF_SECURITY=y
+CONFIG_MBEDTLS=y
+CONFIG_MBEDTLS_ENABLE_HEAP=y
+CONFIG_MBEDTLS_SSL_KEEP_PEER_CERTIFICATE=n
+CONFIG_MBEDTLS_SSL_SERVER_NAME_INDICATION=y
+CONFIG_TLS_CREDENTIALS=y
+CONFIG_TLS_CREDENTIALS_BACKEND_VOLATILE=y
+CONFIG_MBEDTLS_RSA_C=y
+
+# Stack sizes
+CONFIG_NET_CONNECTION_MANAGER_MONITOR_STACK_SIZE=2048
+CONFIG_NET_TX_STACK_SIZE=2048
+CONFIG_NET_RX_STACK_SIZE=2048
+CONFIG_NET_MGMT_EVENT_STACK_SIZE=4096
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=3100
+CONFIG_NRF70_IRQ_WQ_STACK_SIZE=1024
+CONFIG_NRF70_BH_WQ_STACK_SIZE=2300
+CONFIG_SHELL_STACK_SIZE=4400
```

Comment:
please note that with latest WI-Fi the stack sizes have increased (we have backlog item to optimize) but these might need increase as well, you can always use https://github.com/nrfconnect/sdk-nrf/blob/main/samples/wifi/shell/prj.conf#L53 as reference as this would be kept up to date. I am about to raise a PR to sync other samples, but I will leave `net` sample updates for this PR.

Reply:
FYI, Wi-Fi samples PR https://github.com/nrfconnect/sdk-nrf/pull/18227